src/custom-content-type.ts

```diff
@@ -47,9 +48,32 @@ function buildVDomHandler() {
   }
 }
 
+function initVue(initialContent: VNode[], root: Vue) {
+  const blacklist = [
+    '_base',
+    'beforeCreate',
+    'beforeDestroy',
+    'beforeMount',
+    'computed',
+    'destroyed',
+    'mounted',
+    'render',
+  ];
```

Comment:
I'm not sure about this part here. Basically I opened a random Vue application, ran `console.log(Object.keys(vm.$root.$options))` and added all probably-should-not-be-passed-on options to the blacklist here.

My original idea was to create a new dummy Vue instance here and then just keep the difference, but not all lifetime event handlers would have been removed. And some options (e.g. `components`) would be too eagerly removed.

Reply:
Hmm, I'm not too knowledgable on Vue internals, I'm not sure

Reply:
This is pretty much what I suggested in my comment on #105 but looking at the code it feels a bit brittle. I mean absolutely no offense to your implementation @kgilden but at heart this is a workaround: The real issue is that we create a new Vue instance.
But this PR surely will ease some of the issues devs are facing currently.

Reading the [Vue docs](https://vuejs.org/v2/api/#Options-Lifecycle-Hooks), I would add some more lifecycle hooks to the list (which should be renamed to `ignorelist` IMHO):
- created
- beforeUpdate
- updated
- activated
- deactivated
- errorCaptured

Reading the docs further I wonder, if we couldn't use the [`parent`](https://vuejs.org/v2/api/#parent ) option when creating the new Vue instance for the events. I have never used this before, but it sounds like it could resolve this issue in a less hackish way.

Sorry for just smart-assing and not providing any actual code...

Reply:
@rangermeier no worries! Actually `parent` seems perfect. Thanks for bringing it up. Looks like it works great and there is no need to hack with options. Even Vue dev tools started listing the components, which is awesome for debugging.

Thanks!

Reply:
That's great. This means, `$emit`ting from the slot should also work. Thanks for working on this!